src/constants.ts

```diff
@@ -174,20 +175,21 @@ export const ERROR_SENDING_DIFFS = {
 
 export const STATUS_BAR_MSGS = {
 	DEFAULT: ' CodeSync ✅',
-	AUTHENTICATION_FAILED: ' CodeSync ❌, Click to authenticate!',
-	ACCOUNT_DEACTIVATED: ' CodeSync ❌, Click to reactivate your account!',
-	SERVER_DOWN: ' CodeSync ❌, Offline',
+	WAITING_FOR_LOGIN: ' CodeSync $(loading~spin) Waiting for Login confirmation...',
```

Comment:
Is one space after inverted comma deliberate?

Reply:
Yes, this is status bar msg that appears at the bottom of VSCode. So to add a little left-margin, we add a space if user has installed some other extensions too